{
  "uid" : "88fd1119545fa560",
  "name" : "test_refund_phone",
  "fullName" : "testcases.admin_test.test_refund_00001.TestAdminRefund#test_refund_phone",
  "historyId" : "17c62639b42ede51b2bd6cb0bcdc3fe2",
  "time" : {
    "start" : 1649300158164,
    "stop" : 1649300160848,
    "duration" : 2684
  },
  "description" : "该用例是针对获取单个用户信息接口的测试",
  "descriptionHtml" : "<p>该用例是针对获取单个用户信息接口的测试</p>\n",
  "status" : "passed",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "description" : "该用例是针对获取单个用户信息接口的测试",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "358214769529d5cc",
      "name" : "log",
      "source" : "358214769529d5cc.txt",
      "type" : "text/plain",
      "size" : 2871
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "退款模块的测试"
  }, {
    "name" : "epic",
    "value" : "针对单个接口的测试"
  }, {
    "name" : "story",
    "value" : "用例--通过手机号退款"
  }, {
    "name" : "parentSuite",
    "value" : "testcases.admin_test"
  }, {
    "name" : "suite",
    "value" : "test_refund_00001"
  }, {
    "name" : "subSuite",
    "value" : "TestAdminRefund"
  }, {
    "name" : "host",
    "value" : "tongchunyan.local"
  }, {
    "name" : "thread",
    "value" : "60055-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testcases.admin_test.test_refund_00001"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ {
    "name" : "点击，跳转到对应用例的链接地址",
    "url" : "https://www.cnblogs.com/wintest",
    "type" : "test_case"
  }, {
    "name" : "点击，跳转到对应BUG的链接地址",
    "url" : "https://www.cnblogs.com/wintest",
    "type" : "issue"
  } ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "2fd582416314b893",
      "status" : "broken",
      "statusDetails" : "requests.exceptions.SSLError: HTTPSConnectionPool(host='testapi.pookie.com.cn', port=443): Max retries exceeded with url: /adm/order/refund (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1056)')))",
      "time" : {
        "start" : 1649300141893,
        "stop" : 1649300142052,
        "duration" : 159
      }
    } ],
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "88fd1119545fa560.json",
  "parameterValues" : [ ]
}